utils/events/chat_helper_ticket_sys.py

```diff
@@ -1314,6 +1314,16 @@ async def sendCHTKTView(self, interaction: discord.Interaction):
             > {Emoji.ss_arrow} Send a picture of your assignment title in your direct messages as per the bot instructions.""",
             view=TicketButton(self.bot),
         )
+    
+    @app_commands.command(
+        name="title", description="Prompts for assignment title."
+    )
+    @app_commands.guilds(MainID.g_main)
+    @slash_is_bot_admin()
```

Comment:
`@slash_is_bot_admin()`
- Although this needs to be removed since not all chat helpers will have permit 1 on the bot. Any `@slash_is_bot_admin()` or equivalent decorator in `core/checks.py` checks if the user has a permit in the bot. Permits are usually reserved for executive plus so regular staff members won't be able to use this command with this specific check.

Reply:
I see, thanks!